packages/spor-icon-react/bin/generate.ts

```diff
@@ -110,6 +110,7 @@ async function generateComponent(iconData: IconData) {
       titleProp: false,
       svgProps: {
         role: "img",
+        "aria-hidden": "true",
```

Comment:
Denne kommer til å spare oss for endel arbeid - den gjør nemlig ikoner "usynlige" for skjermlesere by default (cc @heisand som foreslo det for lenge siden). Om man mot formodning vil at de skal vises, kan man bare sette `aria-hidden={false}` og heller gi dem en spesifikk `aria-label`.